<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="5" parallel="tests" name="Error Validation Test(failed)" verbose="0">
    <classes>
      <class name="Ahmad.PersonalFramework.test.ErrorValidation">
        <methods>
          <include name="ProductErrorValidation"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- Ahmad.PersonalFramework.test.ErrorValidation -->
    </classes>
  </test> <!-- Error Validation Test(failed) -->
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)" verbose="0">
    <classes>
      <class name="Ahmad.PersonalFramework.test.SubmitOrderTest">
        <methods>
          <include name="tearDown"/>
          <include name="submitOrder"/>
          <include name="OrderHistoryTest"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- Ahmad.PersonalFramework.test.SubmitOrderTest -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
